Jenkinsfile

```diff
@@ -1,268 +0,0 @@
-DOCKER_IMAGE_NAME = 'gcr.io/organic-storm-201412/fetch-ledger-develop:v0.4.2'
```

Comment:
I assume this was removed by mistake!

Reply:
No, I removed this file temporary to prevent CI build of this PR until it is draft. It is a workaround )) I'll restore it for the final review.